src/gql/donations.d.ts

```diff
@@ -163,3 +163,12 @@ export type BankTransactionsUploadImage = {
 export type BankTransactionsFileFormData = {
   bankTransactionsFileId: string
 }
+
+export type BankImportStatus = {
+  status: string
```

Comment:
In the backend the variable name is also status (https://github.com/podkrepi-bg/api/blob/8590c9879aa066bb10b9a31e52e02f119b5072c9/apps/api/src/bank-transactions-file/dto/bank-transactions-import-status.dto.ts#L9). If you think that `importStatus` would be more appropriate I will change it in the both places

Reply:
well, I meant that on the backend the type of the field status is ImportStatus enum not a string as here on the frontend. I see you are just showing it to UI and there is no if conditions on it, so it can stay like this. However the minute someone writes "if status ==="SUCCESS" we will need to turn that into an enum.

Reply:
Ahh I see. I added importStatus as you suggested, so it will be more convenient in the future.